src/App.tsx

```diff
@@ -1,7 +1,25 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  message: string;
+}
+
+export class App extends React.Component {
+  state: Readonly<State> = {
+    message: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (event: KeyboardEvent) => (
+      this.setState({ message: `The last pressed key is [${event.key}]` })
+    ));
+  }
```

Comment:
i think its good to add separate method for this function

```suggestion
 handleKeyUp = (event: KeyboardEvent) => (
      this.setState({ message: `The last pressed key is [${event.key}]` })
    );

  componentDidMount() {
    document.addEventListener('keyup', this.handleKeyUp);
  }
```